arm-insights/2015-04-01/swagger/insightsManagementClient.json

```diff
@@ -0,0 +1,2119 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "InsightsManagementClient",
+    "version": "2015-04-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/microsoft.insights/alertrules/{ruleName}/incidents/{incidentName}": {
+      "get": {
+        "description": "Gets an incident associated to an alert rule",
+        "tags": [
+          "Alert"
+        ],
+        "operationId": "AlertRuleIncidents_Get",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "ruleName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the rule."
+          },
+          {
+            "name": "incidentName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the incident to retrieve."
+          },
+          {
+            "name": "api-version",
```

Comment:
If I use the global parameter, it would need to be the same for all the operations. It is not the same, so I need a way to use a different value. A way to do this is by declaring it as not required and giving it a default value, i.e. every call has a api-version, and it is the correct value, the users don't need to call us to ask what the correct value is, and they can still send a value if they want.